homeworks/hw13/files/hw13_solution.py

```diff
@@ -0,0 +1,74 @@
+class StudentManager:
+    def __init__(self, filename="students.txt"):
+        self.filename = filename
+        self.students_data = []
+
+    def create_students_file(self, text: str) -> None:
+        with open(self.filename, "w", encoding="utf-8") as file:
+            file.write(text)
+
+    def read_students_file(self):
+        self.students_data = []
+        groups = {}
+
+        with open(self.filename, "r", encoding="utf-8") as file:
+            for line in file:
+                try:
+                    name, group, grade = line.strip().split(",")
+                    name = name.strip()
+                    group = group.strip()
+                    grade = int(grade.strip())
+
+                    # pylint: disable=unused-variable
+                    student = (name, group, grade)
+                    self.students_data.append(student)
```

Comment:
It looks like you found a trick to mute pylink with `# pylint: disable=<>`

May it be here like this???
```suggestion
                    self.students_data.append((name, group, grade))
```